spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/web/reactive/ReactiveWebServerFactoryAutoConfigurationTests.java

```diff
@@ -63,6 +68,7 @@
  * @author Brian Clozel
  * @author Raheela Aslam
  * @author Madhura Bhave
+ * @author Chris Bono
```

Comment:
At what level of update to a file are we supposed to add our name as `@author`? I am really not trying to splat my name on every single file - just not sure what the recommended theshold is.

Reply:
Thanks for asking. Please add your name only if you've significantly changed the class. Adding a property doesn't qualify as such. Polishing either. This one is legit as you're adding a number of tests.